<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyRoomForm_template</class>
 <widget class="QDialog" name="ModifyRoomForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="The name of the modify room dialog">Modify room</string>
  </property>
  <layout class="QGridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="capacityTextLabel">
       <property name="text">
        <string>Capacity</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="capacitySpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Building</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="buildingsComboBox"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="nameTextLabel">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
